modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx

```diff
@@ -62,6 +63,15 @@ A note to the reader:
   - [Preview](#preview)
   - [Labs](#labs)
 
+## Canvas Tokens
+
+In v11, all the components listed in this guide have started using our new
+[Canvas Tokens Web](https://workday.github.io/canvas-tokens/?path=/docs/docs-getting-started--docs).
+In v10, we provided fallbacks, in v11 you must add `@workday/canvas-tokens-web` to ensure our
```

Comment:
Should we explain a little about the fallbacks?

```suggestion
In v10, we provided token fallbacks so that a component would not be missing a token/value. In v11 you must add `@workday/canvas-tokens-web` to ensure our
```